x/commerciokyc/keeper/querier.go

```diff
@@ -58,6 +60,26 @@ func queryGetInvites(ctx sdk.Context, _ []string, keeper Keeper, legacyQuerierCd
 	return bz, nil
 }
 
+// queryGetMembership allows to retrieve the current membership of a user having a specified address
+func queryGetInvite(ctx sdk.Context, path []string, keeper Keeper, legacyQuerierCdc *codec.LegacyAmino) (res []byte, err error) {
+	address, err2 := sdk.AccAddressFromBech32(path[0])
```

Comment:
After removing the named return variable `err` this name can be used instead of `err2`